tests/PHPUnit/Fixtures/UITestFixture.php

```diff
@@ -322,7 +322,11 @@ public static function getLocalTestSiteUrl()
     private function addNewSitesForSiteSelector()
     {
         for ($i = 0; $i != 8; ++$i) {
-            self::createWebsite("2011-01-01 00:00:00", $ecommerce = 1, $siteName = "Site #$i", $siteUrl = "http://site$i.com");
+            self::createWebsite("2011-01-01 00:00:00", $ecommerce = 1, $siteName = "Site #$i",
+                $siteUrl = "http://site$i.com", 1, null, null,
+                null, null, 0,
+                implode(',', [$this->xssTesting->forTwig('excludedParameter'),
+                              $this->xssTesting->forAngular('excludedParameter'),'sid']));
```

Comment:
Not sure if it would be needed to add them for each site. Having the globally would be enough for the tests. But besides of that, the new XSS tests need to be added to https://github.com/matomo-org/matomo/blob/3565da78bff718f201247fc2f692c0be29220fde/tests/PHPUnit/Framework/XssTesting.php#L104-L108

Reply:
The specification for this issue changed to only use site excluded parameters and ignore global excluded parameters, that's why I've added them to each test site. I've added the missing XSS expected entries, thanks.